package.json

```diff
@@ -181,6 +181,30 @@
   "pnpm": {
     "overrides": {
       "usocket": "_EXCLUDED_"
+    },
+    "peerDependencyRules": {
+      "allowedVersions": {
+        "esbuild-sass-plugin>esbuild": "^0.16.7",
```

Comment:
`esbuild-sass-plugin´ requires v0.18.0 but the upgrade path contains quite a lot of breaking changes that have to be handled.

Reply:
please add TODO comments in the code - once this PR is closed, we will lose track of such info

Reply:
`package.json` doesn't allow comments unfortunately, where would you like me to add this?